mindsdb/__main__.py

```diff
@@ -44,6 +44,10 @@ def close_api_gracefully(apis):
 
     config = Config(os.environ['MINDSDB_CONFIG_PATH'])
 
+    disable_telemetry = os.getenv('CHECK_FOR_UPDATES').lower() in ['0', 'false']
+    if disable_telemetry:
+        print('\n ✓ telemetry is disabled \n')
```

Comment:
We should be using the `TELEMETRY_FILE` here (see in utils.py -- name is `telemtry.lock`

The logic should be:

On start, if `telemtry.lock` file is present => set CHECK_FOR_UPDATES to 0 (using the function in uitls.py... there should be one)
If `CHECK_FOR_UPDATES` is 0, false, False ... etc => create the `telemtry.lock` file (using the function in utils.py)

Also, we should be re-updating the telemtry value in the gui every time `telemtry.lock` is changed, this is fairly easy since it gets changed with a function in utils.py, so just updated the `index.html` when that happens.

Reply:
done